bolt-sidecar/src/primitives/commitment.rs

```diff
@@ -156,6 +156,19 @@ impl InclusionRequest {
         true
     }
 
+    pub fn validate_min_priority_fee(&self, min_priority_fee: u128) -> bool {
+        for tx in &self.txs {
+            if tx
+                .max_priority_fee_per_gas()
+                .is_some_and(|max_priority_fee| max_priority_fee < min_priority_fee)
```

Comment:
What if in all the `txs` the `max_priority_fee_per_gas` is `None`? That is the case for legacy transactions and should be handled carefully by looking at the `gas_price` instead.